lib/chef/provider/aws_dhcp_options.rb

```diff
@@ -1,7 +1,21 @@
 require 'chef/provisioning/aws_driver/aws_provider'
+require 'chef/provisioning/aws_driver/tagging_strategy/ec2'
 require 'retryable'
 
 class Chef::Provider::AwsDhcpOptions < Chef::Provisioning::AWSDriver::AWSProvider
+
+  def converge_tags
```

Comment:
If we like this idea I'll DRY it up by (probably) putting it into a module.  But this is how a provider creates a strategy and the AWS Tagger instance.